docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,163 @@
+    -   <!--NOTE Limitation from Bitbucket integration V1-->At the moment, we're only collecting new data, so you won’t see any metrics after completing the integration setup. We'll automatically import historical data and backfill your charts within a few weeks.
+
+<!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse integrates with Bitbucket Cloud to receive data about changes and deployments, necessary to calculate the metrics:
+
+-   [Lead time for changes](../metrics/accelerate.md#lead-time-for-changes)
+
+-   [Deployment frequency](../metrics/accelerate.md#deployment-frequency)
+
+## Setting up the Bitbucket integration
+
+!!! note
+    -   The Bitbucket integration connects to a specific [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). You must be the **owner** of that Bitbucket workspace to proceed with the integration setup.
+    -   [See below](#bb-permissions) the permissions that Pulse requires from your Bitbucket account.  
+
+To set up the Bitbucket integration:
+
+1.  On Pulse, [expand **Integrations** and select **Bitbucket**](https://app.pulse.codacy.com/integrations/bitbucket){: target="_blank"}.
+
+1.  Click **Install Bitbucket App** and follow the instructions on the Bitbucket UI to install the app on your account.
+
+    ![Installing the Pulse Bitbucket App](images/bb-installing.png)
+
+1.  Wait until you get a confirmation that Pulse successfully connected the integration to Bitbucket.
+
+    ![Pulse Bitbucket integration set up successfully](images/bb-ok.png)
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+1.  Choose the workspace that you want to connect the integration to. You can only select a workspace where you have owner permissions.
+
+    ![Choosing the Bitbucket workspace](images/bb-workspace.png)
+
+1.  Click **Complete setup**.
+
+    If there is an error please [contact support](mailto:pulsesupport@codacy.com).
+
+## Automatic deployment detection using merged pull requests {: id="deployment-detection-merged-pr"}
+
+!!! note
+    <!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse uses your merged pull requests to automatically detect your deployments. Other strategies to detect deployments will be available with a future version of Pulse.
+
+-   Pulse considers a deployment every merged pull request that **targets the default branch** of the repository.
+-   The deployment date is the timestamp when the corresponding pull request is merged.
+-   The set of changes in a deployment is the list of commits in the corresponding pull request. Pulse correctly tracks your changes even if you [squash or fast-forward the commits when merging the pull request](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Merge-strategies), since Pulse processes all the original commits before any changes to the Git history.
```

Comment:
If 'Squash and merge' is used for Bitbucket, Pulse will not produce accurate accelerate metrics (the only metrics available for V1).

We might be able to overcome this but we don't have time to investigate and implement it for V1, so it will be left for later (see https://codacy.atlassian.net/browse/PUL-1525).

Reply:
See https://github.com/codacy/pulse-user-docs/pull/160#discussion_r1053142215